OxfordBodleian/Juel-Jensen/BDLaethf33.xml

```diff
@@ -514,16 +514,16 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                             <dim unit="leaf">1</dim>
                                             <locus from="1r" to="1v"/>
                                             1, stub after 1
```

Comment:
I think it would theoretically be more precise to give the cut off folio and imaginary folio number (1a or so) and describe this quire as a bifolio, which it is. However, this would lead to an error due to the numbering. So staying with the current suboptimal encoding and describing the situation in condition also seems legitimate to me.